cmd/create/cluster/cmd.go

```diff
@@ -3093,20 +3093,20 @@ func run(cmd *cobra.Command, _ []string) {
 			oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{clusterName, mode, ""})
 		} else {
 			output := ""
-			if cluster.AWS().STS().OidcConfig().Reusable() {
-				output = "When using reusable OIDC Config and resources have been created " +
-					"prior to cluster specification, this step is not required."
-			}
-			rolesCMD := fmt.Sprintf("rosa create operator-roles --cluster %s", clusterName)
-			if permissionsBoundary != "" {
-				rolesCMD = fmt.Sprintf("%s --permissions-boundary %s", rolesCMD, permissionsBoundary)
-			}
-			oidcCMD := "rosa create oidc-provider"
-			oidcCMD = fmt.Sprintf("%s --cluster %s", oidcCMD, clusterName)
-			output += "\nRun the following commands to continue the cluster creation:\n\n"
-			output = fmt.Sprintf("%s\t%s\n", output, rolesCMD)
-			output = fmt.Sprintf("%s\t%s\n", output, oidcCMD)
-			r.Reporter.Infof(output)
+			if !cluster.AWS().STS().OidcConfig().Reusable() {
```

Comment:
It is not guaranteed that the user has created the resources prior when using a reusable oidc configuration